website/blog/2022-01-21-release.md

```diff
@@ -0,0 +1,184 @@
+---
+slug: release-0.20
+title: "New in Wave 0.20"
+author: Martin Turoci
+author_title: Software Engineer @ H2O.ai
+author_url: https://github.com/mturoci
+author_image_url: https://avatars.githubusercontent.com/u/64769322?v=4
+image: /img/wave-type-yellow-1100x400.png
+
+tags: [release]
+---
+
+It's been a long time since the last `0.19` release, but the wait was worth it. The newest Wave includes plenty of extremely useful features that aim to ease the development process and make your Wave apps more appealing. Let's explore the biggest of them.
+
+<!--truncate-->
+
+## Bundled Wave server
+
+Until the `0.20` version, developers were required to manually download Wave server (waved) from [our Github releases page](https://github.com/h2oai/wave/releases), extract and run before running the app. This step is now eliminated and all you need is to run `pip install h2o_wave`. After successful install, run `wave run your-app` and the Wave server should be started automatically.
+
+```shell
+(venv) $ wave run
+```
+
+Note that this simplification is aimed for development only. For production deployments, you might want to pass additional configuration options to waved (not possible with `wave run`) or even switch your python ASGI server (see [deployment docs](/docs/deployment#deploying-wave-apps)).
+
+If you for, some reason, do not want to start `waved` during `wave run`, you can turn it off via `H2O_WAVE_NO_AUTOSTART` env variable.
+
+All the examples and Tour app were previously included in the release. This is not the case with `pip install` though. If you would still like to have all the examples accessible locally, just run `wave fetch` to download examples, demos, and the interactive tour.
+
+```shell
+(venv) $ wave fetch
+```
+
+You should now have access to 200+ examples locally.
+
+```
+Fetching examples and related files. Please wait...
+Downloading https://github.com/h2oai/wave/releases/download/v0.19.0/wave-0.19.0-linux-amd64.tar.gz
+Extracting...
+
+All additional files downloaded and extracted successfully!
+Examples and tour............. /home/elp/git/wave/py/wave/examples
+Demos and layout samples...... /home/elp/git/wave/py/wave/demo
+Automated test harness........ /home/elp/git/wave/py/wave/test
+Wave daemon for deployments... /home/elp/git/wave/py/wave
+```
+
+## Base URL support
+
+By default, Wave apps are all registered on `/` root path. The location your app listens on depends on the route you specify in `@app` or `site['']`. However, if you want to namespace apps or just facing complex deployment, you might want to configure it from a single place rather than on per-app basis.
```

Comment:
```suggestion
By default, Wave apps are all hosted at the root path `/`. This path can be customized by specifying a route via, say, `@app('/foo')` or `site['/foo']`. Starting with v0.20, you can additionally specify a base URL, so that all the apps or sites hosted by a Wave server are served at a path other than `/`. 

Using this feature, specifying a base URL, say `/my/apps`, will make your apps available at `/my/apps/app1`, `/my/apps/app2`, and so on.
```